src/lineChartD3.ts

```diff
@@ -365,25 +365,25 @@ export const lineChartD3 = ((): IChartAdaptor => {
      * Iterates ove data and updates area fills
      */
     drawAreas(data: Array<ILineChartDataSet<any>>) {
-      const { axis, height } = this.props;
+      const { axis, height } = props;
       const yAxisWidth = getYAxisWidth(axis);
       const xAxisHeight = getXAxisHeight(axis);
       const thisArea = (curveType) => area()
         .curve(curveType)
-        .x((d: any) => this.x(d.x) + yAxisWidth)
-        .y0((d) => height - xAxisHeight)
-        .y1((d: any) => this.y(d.y));
+        .x((d: any) => xScale(d.x) + yAxisWidth)
+        .y0((d) => Number(height) - xAxisHeight)
```

Comment:
just a curiosity comment, how come d doesn't have to be typed on this line, but it does above?